remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RulesImpl.java

```diff
@@ -112,6 +119,39 @@ public void deleteRuleById(String folderNodeId, String ruleSetId, String ruleId)
         ruleService.removeRule(folderNodeRef, rule);
     }
 
+    @Override
+    public RuleSetLink linkToRuleSet(String folderNodeId, String linkToNodeId) {
+
+        RuleSetLink ruleSetLink = new RuleSetLink();
+        final NodeRef folderNodeRef = validator.validateFolderNode(folderNodeId,false);
+        final NodeRef linkToNodeRef = validator.validateFolderNode(linkToNodeId, false);
```

Comment:
The linkToNodeId is allowed to be a folder or a rule set, but I think this code only copes when it's a folder. It's probably easiest to create a new sub task where we'll add support for linking using a rule set noderef.